src/apps/experimental/components/library/HomeSectionsView.tsx

```diff
@@ -0,0 +1,127 @@
+import { useGetHomeSectionsWithItems } from 'hooks/useFetchItems';
+import Loading from 'components/loading/LoadingComponent';
+import React, { FC, useMemo } from 'react';
+import SectionContainer from './SectionContainer';
+import globalize from 'scripts/globalize';
+import { SectionType } from 'types/sections';
+import { BaseItemDto } from '@jellyfin/sdk/lib/generated-client';
+import { HomeSectionType } from 'types/homeSectionType';
+import escapeHTML from 'escape-html';
+import { CardShape } from 'utils/card';
+
+type HomeViewProps = {
+    views: BaseItemDto[] | undefined;
+    view?: BaseItemDto;
+    sectionType: HomeSectionType;
+};
+
+const HomeSectionsView: FC<HomeViewProps> = ({
+    view,
+    views,
+    sectionType
+}) => {
+    // Identify the section type for the query
+    const sectionTypeQuery = useMemo(() => {
+        if (sectionType === HomeSectionType.NextUp) {
+            return [SectionType.NextUp];
+        }
+
+        if (sectionType === HomeSectionType.LatestMedia && view?.CollectionType === 'movies') {
+            return [SectionType.LatestMovies];
+        }
+
+        if (sectionType === HomeSectionType.LatestMedia && view?.CollectionType === 'tvshows') {
+            return [SectionType.LatestEpisode];
+        }
```

Comment:
you are missing collection type like books, photos, musics...

Reply:
ok, I don't have those in my library I need to add some and figure that out.

Reply:
@bu3alwa if you allow me, I made some changes I will push the change or I will create a PR

Reply:
added you to the repo

Reply:
@bu3alwa here is the change I made  https://github.com/grafixeyehero/jellyfin-web/commits/convert-home-page-to-react/

Reply:
I added some comments on them. Let me know what you think?

Reply:
@bu3alwa updated

Reply:
are you pushing it to this branch? or should I be doing something to it?

Reply:
here https://github.com/bu3alwa/jellyfin-web/pull/1